app/static/js/bonnerManagement.js

```diff
@@ -69,7 +72,30 @@ function addRequirement() {
     addRequirementsRowHandlers()
     newRow.find("input").focus()
 }
-
+function addCohort(){
```

Comment:
Some additional comments in this function could be great

Reply:
Done!